website/docs/developer/research/research-orca-delete-functionality.md

```diff
@@ -103,6 +123,26 @@ Additional logging should be collected during the deletion process to keep recor
 
 Cloudwatch logs and cloudtrail can be used to collect logging information. In addition, S3 server access logging can be used at the bucket level which would provide detailed records for the requests that are made to a bucket. The log information can be useful in security and access audits. AWS will collect access log records, add the records in log files, and then upload log files to the target bucket as log objects. There is no extra charge for using the service except the cost for storing the log files in S3.
 
+### Initial ORCA Delete Functionality Architecture
+
```

Comment:
Move this up to before you talk about the Lambda and API gateway stuff. Think about your headings (H1-H4) and potentially nest these better to show relationships. I would think I would make this an H2 heading and nest Lambda and other architecture and delete know how headings under this.

Reply:
moved at the top